spec/fixtures/responses/queries/freaks/freaks_pagination.json

```diff
@@ -11,85 +11,15 @@
         },
         "edges": [
           {
-            "node": {
-              "firstName": "Ion",
-              "lastName": "doe",
-              "description": "a simple freak",
-              "email": "freak@gmail.com",
-              "photo": {
-                "id": "1",
-                "uri": "https://bit.ly/3y7afHO"
-              },
-              "skills": [
-                {
-                  "id": "1",
-                  "name": "Project management"
-                },
-                {
-                  "id": "2",
-                  "name": "AWS Architect"
-                }
-              ],
-              "projects": [
-                {
-                  "id": "1",
-                  "name": "EPIX",
-                  "description": "An American premium cable and satellite TV network.",
-                  "logoUrl": {
-                    "id": "1",
-                    "uri": "https://bit.ly/3rB9Yu9"
-                  },
-                  "freaks": [
-                    {
-                      "id": "1",
-                      "firstName": "Ion"
-                    },
-                    {
-                      "id": "1",
-                      "firstName": "Gheorghe"
-                    }
-                  ],
-                  "technologies": [
-                    {
-                      "id": "2",
-                      "name": "Ruby",
-                      "description": "A popular backend language."
-                    }
-                  ]
-                }
-              ],
-              "role": {
-                "id": "1",
-                "name": "Founder"
-              },
-              "level": {
-                "id": "3",
-                "name": "Advanced"
-              },
-              "norm": {
-                "id": "1",
-                "name": "full-time"
-              },
-              "specialities": [
-                {
-                  "id": "2",
-                  "name": "Backend"
-                }
-              ],
-              "technologies": [
-                {
-                  "id": "1",
-                  "name": "GraphQl"
-                },
-                {
-                  "id": "2",
-                  "name": "Ruby"
-                }
-              ]
-            }
+            "node": null
```

Comment:
Do we really expect the response to be `null` instead of it returning data? :))

Reply:
Solved.